src/test/java/org/broadinstitute/hellbender/engine/FeatureSupportInVariantsWalkersIntegrationTest.java

```diff
@@ -0,0 +1,51 @@
+package org.broadinstitute.hellbender.engine;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.tools.examples.ExampleVariantWalker;
+import org.broadinstitute.hellbender.utils.test.ArgumentsBuilder;
+import org.testng.annotations.Test;
+
+import java.io.File;
+
+public final class FeatureSupportInVariantsWalkersIntegrationTest extends CommandLineProgramTest {
```

Comment:
After a second look at this, these tests are not really for "Feature support" specifically. Recommend that you rename this class to just `VariantWalkerIntegrationTest`, and rename `FeatureSupportInReadWalkersIntegrationTest` back to `FeatureSupportIntegrationTest`